package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "scroll-timeline-polyfill",
+  "version": "1.0.0",
+  "description": "A polyfill for scroll-driven animations on the web via ScrollTimeline",
+  "source": "src/index.js",
+  "main": "dist/scroll-timeline.js",
+  "scripts": {
+    "build": "microbundle -f iife",
+    "watch": "microbundle watch -f iife",
+    "dev": "run-all \"serve\" \" npm run watch \" ",
+    "deploy": "npm run build && ./deploy.sh && gh-pages build",
+    "test:unit": "jest",
+    "test:watch": "jest --watchAll",
+    "test:coverage": "jest --collect-coverage",
+    "format": "prettier --write \"src/**/*.js\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/flackr/scroll-timeline.git"
+  },
+  "files": [
+    "src",
+    "dist"
+  ],
+  "keywords": [
+    "waapi",
+    "scrolltimeline",
+    "parallax",
+    "animations",
+    "web",
+    "animations"
+  ],
+  "author": "",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/flackr/scroll-timeline/issues"
+  },
+  "babel": {
+    "env": {
+      "test": {
+        "plugins": [
+          "@babel/plugin-transform-modules-commonjs"
+        ]
+      }
+    }
+  },
+  "homepage": "https://github.com/flackr/scroll-timeline#readme",
+  "devDependencies": {
```

Comment:
We now have 2K transitive package dependencies. 😮 

Almost all of these are dev dependencies but this was a bit just surprising to me. I guess bundling a whole web-server/testing framework (jest) and compiler/bundler (babel) does this. So I am not sure if there is a way to avoid this.

Reply:
hi Majid, we are removing `jest`, `babel` and the existing unit tests, and we will be using `WPT` tests, the WAAPI polyfill is in fact way worse than that: https://github.com/web-animations/web-animations-js/blob/dev/package.json

The new solution I am going to propose to you for testing WPT with polyfills should help reducing those dependencies. 

I look forward to updating the PR very soon & for your review & feedback!